dep/jsoncpp/jsoncpp.cpp

```diff
@@ -1845,7 +1845,7 @@ bool OurReader::decodeNumber(Token& token, Value& decoded) {
     const auto digit(static_cast<Value::UInt>(c - '0'));
     if (value >= threshold) {
       // We've hit or exceeded the max value divided by 10 (rounded down). If
-      // a) we've only just touched the limit, meaing value == threshold,
+      // a) we've only just touched the limit, meaning value == threshold,
```

Comment:
oops, I think this file is excluded by `excludes` -- when I started this work, I worked with a default `excludes` list and let this version of check-spelling regenerate its own excludes -- I only later reapplied most of the excludes.

I'll drop this...